### PR TITLE
feat: update alchemy API keys

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Uses new Alchemy API keys - note there isn't one per chain now hence the unification.

## Issue (if applicable)

closes N/A

## Risk

> High Risk PRs Require 2 approvals

> What protocols, transaction types, wallets or contract interactions might be affected by this PR?

None

## Testing

- Filter network requests by `alchemy` 
- Ensure we get 200 back

### Engineering

- ^

### Operations

- [ ] :checkered_flag: My feature is behind a flag and doesn't require operations testing (yet)

- ^

## Screenshots (if applicable)

- develop

Screenshot 2024-10-17 at 14 33 24

- this diff

Screenshot 2024-10-17 at 14 35 23

### CODE REVIEW

File: .env.base [hunk 1]
```diff
@@ -94,15 +94,8 @@ REACT_APP_COINGECKO_API_KEY=CG-we5Z4KbdYMCgMUVAqDjQMWfc
...
 
-# unstoppable domains resolution
-REACT_APP_ALCHEMY_API_KEY=3DnVf4ZoxBk0W0TZcipqAhmATm7uc1vE
-
-# Alchemy API keys - to be used either with Alchemy SDK or directly with the REST endpoints
-REACT_APP_ALCHEMY_POLYGON_JAYPEGS_API_KEY=OqVQS8M_NzYeV--9xHZjKEIscrZOADzp
-REACT_APP_ALCHEMY_ETHEREUM_JAYPEGS_API_KEY=yh6GF21GaU8wlvEB4wYp7ssXhCU76gOx
-REACT_APP_ALCHEMY_OPTIMISM_JAYPEGS_API_KEY=K03vX5rMZeGmF2V9U-SCsrCeeVnr8qmO
-REACT_APP_ALCHEMY_ARBITRUM_JAYPEGS_API_KEY=DqH3kvkDIq8qwit_G8TM4DFQe5s5TKdq
-REACT_APP_ALCHEMY_BASE_JAYPEGS_API_KEY=MLYU1OYYxF5PCANjGVFs8fIlGNVZAbtO
```
Comment (NeOMakinG) [inline .env.base:105]: Any reasons why we had one key by chain? Did we have rate issues?

**Review by NeOMakinG — APPROVED** [review]
image
Stamping from a functional POV but the multi keys is a bit scary

File: .env.base (hunk 1 above)
Comment (gomesalexandre) [reply .env.base:105]: As mentioned in the description, multiple API keys existed previously (one per chain), but Alchemy now give us the same one for all chains

File: .env.base [hunk 2]
```diff
@@ -94,15 +94,8 @@ REACT_APP_COINGECKO_API_KEY=CG-we5Z4KbdYMCgMUVAqDjQMWfc
 REACT_APP_EXCHANGERATEHOST_BASE_URL=https://api.exchangerate.host
 REACT_APP_EXCHANGERATEHOST_API_KEY=8f7515ffddef9d3e449b45f93108ca4d
 
-# unstoppable domains resolution
-REACT_APP_ALCHEMY_API_KEY=3DnVf4ZoxBk0W0TZcipqAhmATm7uc1vE
```
Comment (gomesalexandre) [inline .env.base:98]: Also, this guy was previously unused

**Pull request merged**